shared/settings/feedback/index.tsx

```diff
@@ -137,9 +137,20 @@ class Feedback extends React.Component<Props, State> {
                 />
               </Kb.Box2>
             )}
-            <Kb.ButtonBar>
-              <Kb.Button fullWidth={true} label="Send" onClick={this._onSendFeedback} waiting={sending} />
-            </Kb.ButtonBar>
+            {!this.props.loggedOut && (
```

Comment:
want logged out feedback page to have centered button, and settings feedback page to have left-justified button